libraries/provider_rackspace_monitoring_service.rb

```diff
@@ -21,9 +21,12 @@ def whyrun_supported?
           action new_resource.package_action
         end
 
+        default_unless['rackspace_monitoring']['create_entity'] = true
+        auto_create = node['rackspace_monitoring']['create_entity'] || new_resource.create_entity ? '--auto-create-entity' : ''
```

Comment:
This was the most flexible way I could think of allowing the auto-create functionality. It can be globally disabled via an attribute (environment-wide) and still selectively enabled on a per-service basis.

Default behavior is to enable auto-create functionality.